src/components/InternalDropdown/index.js

```diff
@@ -207,10 +220,17 @@ const InternalDropdown = forwardRef((props, reference) => {
         return handleChange(rest);
     };
 
+    const handleTabKeyPressed = event => {
+        if (showCheckbox && event.target.tagName !== 'BUTTON') {
+            event.stopPropagation();
+        }
+    };
```

Comment:
what is this used for?

Reply:
This is used to prevent the dropdown to close when clicking the Only button